R/multisynth_class.R

```diff
@@ -86,6 +86,12 @@ multisynth <- function(form, unit, time, data,
                                 time_w = F, lambda_t = 0,
                                 fit_resids = TRUE, eps_abs = eps_abs,
                                 eps_rel = eps_rel, verbose = verbose, long_df = long_df, ...)
+    
+   
+    if("weights" %in% names(msynth)) {
```

Comment:
Ditto on the possibility that there's no weights variable. Does this ever happen?

Reply:
Ditto above.